dev/io.openliberty.mcp.internal/src/io/openliberty/mcp/internal/responses/McpResponse.java

```diff
@@ -19,12 +22,13 @@
 public abstract class McpResponse {
     private String jsonrpc;
     private McpRequestId id;
+    private static final TraceComponent tc = Tr.register(McpResponse.class);
 
     public McpResponse(String jsonrpc, McpRequestId id) {
         if (jsonrpc == null || !jsonrpc.equals("2.0"))
-            throw new IllegalArgumentException("jsonrpc field must be present. Only JSONRPC 2.0 is currently supported");
+            throw new IllegalArgumentException(Tr.formatMessage(tc, "CWMCM0021E.jsonrpc.validation.invalid.version", jsonrpc));
         if (id == null)
-            throw new IllegalArgumentException("id must not be null");
+            throw new IllegalArgumentException(Tr.formatMessage(tc, "CWMCM0026E.jsonrpc.validation.invalid.id.type", id));
```

Comment:
I don't think these need translated, I think they're just internal checks which should never fail unless there's a bug in our code.